codesystems/CodeSystem-England-PeriodType.xml

```diff
@@ -23,9 +23,16 @@
   <concept>
     <code value="Legal" />
     <display value="Legal" />
+    <definition value="Legal period that data can be held." />
   </concept>
   <concept>
     <code value="Operational" />
     <display value="Operational" />
+    <definition value="Operational period that data can be held." />
+  </concept>
+  <concept>
+    <code value="system-effective" />
+    <display value="System Effective" />
+    <definition value="System effective date that data was first added or moved to history on Patient Demographic Service (PDS)." />
```

Comment:
Can we get rid of PDS as this code should be able to be used by any programmes, not just them.

I'm still not certain what the difference is between operational and system-seffective? It feels like they are the same thing said in 2 different ways? Should it be `Historical` going by your description?

Reply:
if we remove PDS from that code, then System-effective date and operational date will become duplicate of each other. As they will both mean a period that data can be held for operation purpose. However, By Sam explanation, System-effective date is specific about when data gets added to PDS. E.g., the start date of someone RelatedPerson (RelatedPerson.period) may not be the same date the data gets added to PDS.

Reply:
Is there a time when we expect `operational` to be used by PDS or another team at the same time as `system-eefective` where they mean something different?